.github/workflows/deploy.yaml

```diff
@@ -200,9 +200,6 @@ jobs:
       - name: 'Verify did endpoint is available'
         run: curl https://${{ steps.runterraform.outputs.did_host }}/.well-known/did.json | jq '.id'
 
-      - name: 'Verify deployed Registry Service is healthy'
```

Comment:
out of curiosity: why this check is removed and no other health check is introduced in its place?

Reply:
This is explained in the notes, the endpoint is not accessible anymore. Created https://github.com/agera-edc/MinimumViableDataspace/issues/212 for F/U implementation